index.js

```diff
@@ -30,6 +30,20 @@ document.body.appendChild(renderer.domElement)
 // Create viewer
 const viewer = new Viewer(renderer)
 
+const textures = []
+
+const loader = new THREE.TextureLoader()
+
+let breakStartTime
+
+for (let i = 0; i < 10; i++) {
+  loader.load('textures/1.16.4/blocks/destroy_stage_' + i + '.png', (texture) => {
+    texture.magFilter = THREE.NearestFilter
```

Comment:
Don't hardcode, get the version from viewer.version

Reply:
Version isn't loaded at this stage. To get the version you need to move this loop into the spawn event handler.

Reply:
Karang: I moved it out.